README.md

```diff
@@ -1,2 +1,3 @@
-# endless-sky.github.io
-Web Page
+# Website for [Endless Sky](https://github.com/endless-sky/endless-sky)
+
+The website for the space trading and combat game [Endless Sky](https://github.com/endless-sky/endless-sky). View it here: https://endless-sky.github.io.
```

Comment:
It seems like GitHub includes the dot on the last link as part of the URL. Can that be fixed? (idk if not then whatever)

Reply:
I'm not sure what you mean:
<img width="757" alt="Screen Shot 2023-10-28 at 5 50 32 PM" src="https://github.com/endless-sky/endless-sky.github.io/assets/125756761/7172ab89-209a-4dde-a712-c87b6d7c0ea7">

But if it's appearing to other people that way, this could make a difference:
```suggestion
The website for the space trading and combat game [Endless Sky](https://github.com/endless-sky/endless-sky). View it here: [https://endless-sky.github.io](https://endless-sky.github.io).
```

Reply:
Try hovering over the link with your mouse

Reply:
Still not seeing anything. Could you send a screenshot of what you're seeing?
<img width="359" alt="Screen Shot 2023-10-29 at 11 05 35 AM" src="https://github.com/endless-sky/endless-sky.github.io/assets/125756761/edbc0b7c-a418-47b0-88a4-9023d78a32a5">

Reply:
oh it's just in the PR, nvm